From f8e468d45df44b947601ceafdc385a15ec24bece Mon Sep 17 00:00:00 2001
From: Idan Kahlon <idan.kahlon@tandemg.com>
Date: Wed, 27 Oct 2021 11:08:22 +0300
Subject: [PATCH] [SW-0] gaudi2: common: avoid nested redefintion

common.h was missing ifndef which may cause
nested redefinition compilation errors

Change-Id: Iecdf75c1840d00ac52e12146b0e44ffd6ddc331c
Signed-off-by: Idan Kahlon <idan.kahlon@tandemg.com>
(cherry picked from commit 5e280bd8f3d39fca7015c25d3ec69a04b5f5ad61)
---

diff --git a/arch/gaudi2/common/common.h b/arch/gaudi2/common/common.h
index 7fdda51..f4c9f0e 100644
--- a/arch/gaudi2/common/common.h
+++ b/arch/gaudi2/common/common.h
@@ -3,6 +3,9 @@
  * Copyright (C) 2021 HabanaLabs Ltd.
  */
 
+#ifndef ARCH_GAUDI2_COMMON_COMMON_H_
+#define ARCH_GAUDI2_COMMON_COMMON_H_
+
 #include <preboot.h>
 #include <limits.h>
 #include <gaudi2_blocks_linux_driver.h>
@@ -171,3 +174,4 @@
 };
 
 void gaudi2_axi_drain_ena(void);
+#endif /* ARCH_GAUDI2_COMMON_COMMON_H_ */
